tests/conftest.py

```diff
@@ -109,26 +110,21 @@ def metrics():
     )
 
 
-@pytest.fixture
+@pytest.fixture(scope="session")
 def remote_addr():
-    return "1.2.3.4"
+    return "192.0.2.1"
 
 
-@pytest.fixture
-def remote_addr_hashed():
-    """
-    Static output of `hashlib.sha256(remote_addr.encode("utf8")).hexdigest()`
-    Created statically to prevent needing to calculate it every run.
-    """
-    return "6694f83c9f476da31f5df6bcc520034e7e57d421d247b9d34f49edbfc84a764c"
+@pytest.fixture(scope="function")
```

Comment:
Shouldn't all of these be scoped to `session`, so they effectively become set once per test suite run?

```suggestion
@pytest.fixture(scope="session")
```